docs/03-policies/security.md

```diff
@@ -181,27 +188,27 @@ Developers and themers working on the site codebase (and committing code to Git)
   - Session hijacking, prevented by using SSL and correct site/session settings.
   - Data disclosure, prevented by carefully setting and testing access control, as well as using SSL as needed.
   - Password guessing attacks, mitigated by using strong passwords.
-- Software that is not licensed under an approved CivicActions open source license may not be used on a project without prior approval from the legal team.
+- Avoid inclusion of any software not licensed under an approved CivicActions open source license on a project without prior approval from the legal team.
```

Comment:
We should keep the original sentence. Can we remove the change for this line so it reads "Software that is not licensed under an approved CivicActions open source license may not be used on a project without prior approval from the legal team."